sdk/storage/azure-storage-queue/azure/storage/queue/aio/_models.py

```diff
@@ -87,7 +103,7 @@ def __init__(self, command, prefix=None, results_per_page=None, continuation_tok
         self.results_per_page = results_per_page
         self.location_mode = None
 
-    async def _get_next_cb(self, continuation_token):
+    async def _get_next_cb(self, continuation_token: str) -> Self:
```

Comment:
Use updated type from sync version of this model.

Reply:
Not sure what this comment means? Sync looks like this for context:
`def _get_next_cb(self, continuation_token: str) -> Self:`
So I think they are already matching? Possible that I ninja-fixed this while you were making the comment 😄

Reply:
Hmmmm I swear at one point we had something different here...

Anyway `Self` is not the correct return type here or for `_extract_data_cb`. Let's make this one `Any` and `_extract_data_cb` below should be `Tuple[Optional[str], List[QueueProperties]]`.

Reply:
Ah, I figured out what happened. I meant to leave this comment on the above `MessagesPaged` class where we did change the return types in sync! So for `MessagesPaged` above we should match what we did for sync.

For this class, `QueuePropertiesPaged`, we should make the update I mentioned above here and in sync.

Sorry for the confusion. A lot to look at in this PR :).

Reply:
No worries 😛 and the GitHub preview always leaves a little bit to be desired...